tasklist/data-generator/src/main/java/io/camunda/tasklist/data/DevDataGeneratorAbstract.java

```diff
@@ -28,9 +29,11 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.DependsOn;
 import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
 import org.springframework.security.crypto.password.PasswordEncoder;
 
+@DependsOn("schemaStartup")
```

Comment:
It somehow feels so weird to depend on something which is not in the same module, like here we now depend on something from the web-app. So I can't use the data generator standalone ? 🤔 But Idk I guess I'm not familiar enough with spring